CONTRIBUTING.md

```diff
@@ -111,9 +119,48 @@ you must run a local docker daemon.
 or alternatively initialize submodules for an existing clone.
 
    `git submodule init`
-   `git submodule update`   
+   `git submodule update`
 
 3. Run the following commands to build, run tests and most static analysis, and
 check formatting:
 
     `./gradlew check`
+
+## Snapshots
```

Comment:
Thanks for the updates! The only thing that comes to mind is that snapshots are also often used by end users, not just contributors. It's probably ok though

Reply:
Yeah, I'm not sure why we'd want to remove this information from the main README.

Reply:
Yeah, I went back and forth on this -- I would assume that releases are "stable" while snapshots may not be. I can add a link to this in the contributing file that way it is accessible.

Reply:
Addressed by: https://github.com/open-telemetry/opentelemetry.io/pull/310